stix_shifter_modules/azure_log_analytics/stix_translation/json/SecurityEvent_from_stix_map.json

```diff
@@ -0,0 +1,171 @@
+            ]
+        }
+    },
+    "domain-name": {
+        "fields": {
+            "value": [
+                "TargetDomainName"
+            ]
+        }
+    },
+    "user-account": {
+        "fields": {
+            "account_login": [
+                "TargetUserName"
+            ]
+        }
+    },
+    "directory": {
+        "fields": {
+            "path": [
+                "HomeDirectory",
+                "HomePath"
+            ]
+        }
+    },
+    "file": {
+        "fields": {
+            "path": [
+                "FilePath"
+            ],
+            "hashes": [
+                "FileHash"
+            ]
+        }
+    },
+    "process": {
+        "fields": {
+            "name": [
+                "ProcessName",
+                "LogonProcessName"
+            ],
+            "parent_ref.name": [
+                "ParentProcessName"
+            ],
+            "command_line": [
+                "CommandLine"
+            ],
+            "pid": [
+                "ProcessId"
+            ]
+        }
+    },
+    "url": {
+        "fields": {
+            "value": [
+                "QuarantineHelpURL"
+            ]
+        }
+    },
+    "x-ibm-finding": {
+        "fields": {
+            "start": [
+                "TimeGenerated"
+            ],
+            "end": [
+                "PreviousTime"
+            ],
+            "code": [
+                "EventID"
+            ],
+            "name": [
+                "EventSourceName"
+            ],
+            "time_observed": [
+                "TimeGenerated"
+            ],
+            "src_ip_ref": [
+                "IpAddress"
+            ],
+            "dst_ip_ref": [
+                "ClientIPAddress"
+            ]
+        }
+    },
+    "x-oca-event": {
+        "fields": {
+            "created": [
+                "TimeGenerated"
+            ],
+            "code": [
+                "EventID"
+            ],
+            "provider": [
+                "Account"
+            ],
+            "url_ref": [
+                "QuarantineHelpURL"
+            ],
+            "process_ref": [
+                "ProcessName"
+            ],
+            "file_ref.hash": [
+                "FileHash"
+            ],
+            "file_ref.path": [
+                "FilePath"
+            ],
+            "file_ref.service": [
+                "ServiceFileName"
+            ],
+            "parent_process_ref": [
+                "ParentProcessName"
+            ],
+            "ip_refs.ip": [
+                "IpAddress"
+            ],
+            "ip_refs.clientIp": [
+                "ClientIPAddress"
+            ]
+        }
+    },
+    "x-azure-security-event": {
+        "fields": {
+            "title": [
+                "Title"
+            ],
+            "text": [
+                "Description",
+                "DeviceDescription"
+            ],
+            "device_name": [
+                "DeviceId"
```

Comment:
the device id and computer seem like it could fit in `x-oca-asset`? https://github.com/opencybersecurityalliance/stix-extensions/blob/main/2.0/x-oca-asset.md